codalab/apps/web/forms.py

```diff
@@ -85,15 +86,8 @@ class Meta:
             'scoring_program_organizer_dataset',
             'phase_never_ends',
             'force_best_submission_to_leaderboard',
-            'scoring_program_docker_image',
```

Comment:
can we just comment these out or something instead?

Reply:
This is now the case, I just believe git is not displaying the commented out code.

Reply:
ty